dandi/metadata/nwb.py

```diff
@@ -72,7 +72,7 @@ def get_metadata(
             path_metadata = df.get_metadata(digest=digest)
             meta["bids_version"] = df.get_validation_bids_version()
             # there might be a more elegant way to do this:
-            if path_metadata.wasAttributedTo is not None:
+            if path_metadata.wasAttributedTo is not None and len(path_metadata.wasAttributedTo) > 0:
```

Comment:
more generic would be then just

```suggestion
            if path_metadata.wasAttributedTo:
```

I think.

Reply:
Sure, it is more generic by abstracting to broader 'truthiness' principles, but less specific to the assertions of that particular variable (technically, we could still encounter failures in the lines below if the attribute is not a list at all)

I leave it to you if you want to be general or specific (feel free to commit suggestion if you prefer)